atlasdb-commons/src/main/java/com/palantir/common/exception/AtlasDbDependencyException.java

```diff
@@ -15,18 +15,61 @@
  */
 package com.palantir.common.exception;
 
-public class AtlasDbDependencyException extends RuntimeException {
-    private static final String EXCEPTION_MESSAGE = "AtlasDB dependency threw an exception.";
+import com.google.errorprone.annotations.CompileTimeConstant;
+import com.palantir.logsafe.Arg;
+import com.palantir.logsafe.Safe;
+import com.palantir.logsafe.SafeLoggable;
+import com.palantir.logsafe.exceptions.SafeExceptions;
+import java.util.List;
+import javax.annotation.Nullable;
 
-    public AtlasDbDependencyException(String msg) {
-        super(msg);
+public class AtlasDbDependencyException extends RuntimeException implements SafeLoggable {
+    private static final String ATLASDB_DEPENDENCY_LOG_MESSAGE = "AtlasDB dependency threw an exception.";
+    private final List<Arg<?>> args;
+    private final String logMessage;
+
+    public AtlasDbDependencyException(@CompileTimeConstant final String logMessage) {
```

Comment:
nit: String doesn't need to be marked final here. We tend not to mark variables as final - there's an internal style guide floating around that's worth reading up!

Reply:
also nit: Don't forget the advice I gave in the call! You don't need one constructor with args and one without, and you should just have the one private constructor at the end setting the fields. See KeyAlreadyExistsException as an example.